### PR TITLE
Most Frequent Users -> most used words dropdown complete

### LINKED ISSUE #97: MostUsedWords by a particular user in the MostActiveUsers component
eg

< profile picture> @WayneRooney 25 tweets - **most frequent words: pal(100), mate(100), ball(11), player(8)…**

currently, just ha a 'Most Frequent Words' placeholder.

### PR DESCRIPTION
**Pull request by VasilyShelkov**
fixes #97 

@Winwardo ready for comments + merge

### CODE REVIEW

File: src/client/results/mostfrequent/users/UserItemsList.js [hunk 1]
```diff
@@ -13,14 +15,17 @@ class UserItemsList extends Component {
...
         <table className="ui very basic table">
           <thead>
             <tr>
               <th className="eight wide"></th>
-              <th className="two wide"></th>
-              <th className="four wide"></th>
-              <th className="two wide"></th>
+              <th className="one wide"></th>
+              <th className="six wide">
+                <i className="trophy yellow icon"></i>
```
Comment (Winwardo) [inline src/client/results/mostfrequent/users/UserItemsList.js:27]: _sigh_

File: src/client/results/mostfrequent/users/UserItemsList.js [hunk 2]
```diff
@@ -70,4 +76,20 @@ let UserItem = ({ dispatch, userInfo }) => (
 );
 UserItem = connect()(UserItem);
 
+const UserItemMostUsedWords = ({ usersMostUsedWords }) => (
+  <div className="ui pointing fluid dropdown">
+    <div className="text"><b>{usersMostUsedWords[0].word}</b> x{usersMostUsedWords[0].count}</div>
```
Comment (Winwardo) [inline src/client/results/mostfrequent/users/UserItemsList.js]: If I'm being nitpicky, this should be <strong> not <b>

File: src/client/tweetAnalysis.js [hunk 3]
```diff
@@ -95,17 +107,21 @@ const categoriseByUser = (posts) => (
  * Given some list of tweets, create a dictionary of how often each word appears.
  * Correctly matches https://t.co/~ URLs, @mentions and #hashtags
```
Comment (Winwardo) [inline src/client/tweetAnalysis.js]: This still says we match t.co links when we do not

File: src/client/tweetAnalysis.js [hunk 4]
```diff
@@ -50,16 +49,29 @@ export const groupedCountWords = (countedWords) => {
...
  */
-export const mostFrequentUsers = (tweets) => (
-  categoriseByUser(tweets).sort((tweetList1, tweetList2) => (
-    tweetList2.posts.length - tweetList1.posts.length
-  ))
-);
+export const mostFrequentUsers = (tweets) => {
+  const topUsers = categoriseByUser(tweets);
+  let topRestUsers = [];
+  if (topUsers.length > 10) {
```
Comment (Winwardo) [inline src/client/tweetAnalysis.js]: Is it worth making this 10 and all others a const somewhere?

File: src/client/results/mostfrequent/users/UserItemsList.js (hunk 2 above)
Comment (VasilyShelkov) [inline src/client/results/mostfrequent/users/UserItemsList.js]: I'm sure you're right, but what's the reason ?

Comment (Winwardo) [inline src/client/results/mostfrequent/users/UserItemsList.js]: Semantics - <strong> says "this is an important element" whereas <b> says "this is a bold element" - the point being that HTML should be declarative, and not actually mention styling in any of the tags.

https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em

File: src/client/tweetAnalysis.js [hunk 5]
```diff
@@ -105,9 +105,9 @@ const categoriseByUser = (posts) => (
 
 /**
  * Given some list of tweets, create a dictionary of how often each word appears.
- * Correctly matches https://t.co/~ URLs, @mentions and #hashtags
- * Punctuation (hyphens, commas, full stops) are counted as spaces.
- * Removes https://t.co/~ links
+ * Removes matched https://t.co/~ URLs
+ * Correctly matches @mentions and #hashtags
+ * unctuation (hyphens, commas, full stops) are counted as spaces.
```
Comment (Winwardo) [inline src/client/tweetAnalysis.js]: "unctuation" hahaha :P

**Pull request merged**